docs/http.md

```diff
@@ -241,12 +241,36 @@ Prefer: respond-async
 }
 ```
 
-it can be canceled with
+It can be canceled with:
 
 ```http
 POST /predictions/abcd1234/cancel HTTP/1.1
 ```
 
-Use of the cancelation API to cancel predictions started "synchronously" (i.e.
-without the `Prefer: respond-async` header) is currently not supported. This may
-change in a future release of Cog.
+Predictions cannot be canceled if they're
+created without a provided `id`
+or synchronously, without the `Prefer: respond-async` header.
+
+If no prediction exists with the provided `id`,
+then the server responds with status `404 Not Found`.
+Otherwise, the server responds with `200 OK`.
+
+When a prediction is canceled,
+Cog raises `cog.server.exceptions.CancelationException`
+in the model's `predict` function.
+This exception may be caught by the model to perform necessary cleanup.
```

Comment:
Passive voice is not something liked by me, however...

I originally had "You", but thought that was ambiguous since the rest of this document is framed with explicit client and server roles.